modules/core/pom.xml

```diff
@@ -76,6 +76,7 @@
     <dependency>
       <groupId>org.apache.thrift</groupId>
       <artifactId>libthrift</artifactId>
+      <version>0.9.1</version>
```

Comment:
This is okay for now, but we should look into shading in a newer version after 1.2.0 is released.